src/constants.js

```diff
@@ -1,26 +1,11 @@
-// TODO: Remove unused constants and add new ones
```

Comment:
Did you add new constants? I mean if you search in the code and find new strings that are not used here, please add them

Reply:
I only removed unused constants.

Reply:
No, problem I'm going to **add new constants** around the code... Please, if you find some test more to add let me know!